testing/sdk_test.go

```diff
@@ -215,45 +219,6 @@ func (s *IntegrationTestSuite) TestLegacyRestErrMessages() {
 func (s *IntegrationTestSuite) testQueryIBCTx(txRes sdk.TxResponse, cmd *cobra.Command, args []string) {
 	val := s.network.Validators[0]
 
-	errMsg := "this transaction cannot be displayed via legacy REST endpoints, because it does not support" +
```

Comment:
I am a bit unsure about removing all this part...

Reply:
did the legacy REST endpoints get removed completely? If so I think this entire file could be deleted (unsure). You should check with the SDK team (these tests are from the SDK but were moved to ibc-go because of the dependency in ibc-go). In general they shouldn't be removed unless the functionality they were testing has been removed (which requires an ACK from the SDK team)

summoning @amaurym, any idea if these tests can be removed now?

Reply:
Ok, thanks, @colin-axner. I will check.

Reply:
I think that they are simply gone-- at least that's my understanding.